app/src/lib/git/status.ts

```diff
@@ -67,9 +67,19 @@ export async function getStatus(
   const result = await spawnAndComplete(
     ['status', '--untracked-files=all', '--branch', '--porcelain=2', '-z'],
     repository.path,
-    'getStatus'
+    'getStatus',
+    new Set<number>([0, 128])
```

Comment:
Is TS smart enough to infer that this is `<number>` without specifying it directly?

Reply:
Actually, yes. Just a habit of mine. Will tidy this up :art: